lib/libutee/tee_api_property.c

```diff
@@ -201,7 +201,7 @@ TEE_Result TEE_GetPropertyAsString(TEE_PropSetHandle propsetOrEnumerator,
 {
 	TEE_Result res;
```

Comment:
While you're at it, please initialize all the locals here.

Reply:
Ok, I will do that in a separate commit

Reply:
I don't think this is a false positive. We can reach the following if statement without `type` being initialized
https://github.com/OP-TEE/optee_os/blob/58d050167425a8513ceee9b8b21878ae0b13ee8f/lib/libutee/tee_api_property.c#L228

Reply:
How? I followed the different functions (and syscalls) and couldn't find the spot.

Reply:
In its analysis, Coverity does not follow the reference to `type` given as an argument to `propget_get_property()`. The analysis stays in the scope of `TEE_GetPropertyAsString()`

Reply:
Ok, I will add it

Reply:
Wait, it looks like `propget_get_property()` can return without initializing `type` no? It would not a false positive in that case.

Reply:
Yes, but it doesn't return TEE_ERROR_SHORT_BUFFER. However, it's a bit hard to tell with 100% certainty so I'm fine with this commit either way.